api/tests/unit/domain/models/CleaCertificationScoring_test.js

```diff
@@ -143,208 +123,101 @@ describe('Unit | Domain | Models | CleaCertificationScoring', function () {
 
     context('reproducibility rate in green zone', function () {
       // eslint-disable-next-line mocha/no-setup-in-describe
-      GREEN_ZONE_REPRO.forEach((reproducibilityRate) =>
-        it(`for ${reproducibilityRate} reproducibility rate, it should obtain certification`, async function () {
-          // given
-          const cleaCertificationScoring = await _buildCleaCertificationScoringWithReproducibilityRate(
-            reproducibilityRate
-          );
-
-          // when
-          const hasAcquiredCertif = cleaCertificationScoring.isAcquired();
-
-          // then
-          expect(hasAcquiredCertif).to.be.true;
-        })
-      );
-    });
-
-    context('reproducibility rate in grey zone', function () {
-      it("for 70 reproducibility rate, it should obtain certification when the pixScore for each certifiable competences is above a floored 75% of Clea corresponding competence's pixScore for at least 75% of them", async function () {
-        // given
-        const cleaCertificationScoring =
-          await _buildCleaCertificationScoringInGreyZoneAndAtLeast75PercentOfCertifiableCompetences();
+      GREEN_ZONE_REPRO.forEach((reproducibilityRate) => {
```

Comment:
Le titre du test est autoportant, supprimer le tableau en haut (faut scroll pour trouver les valeurs)
```suggestion
    [50, 70, 90].forEach((reproducibilityRate) => {
```